CUDADataFormats/HGCal/interface/HGCRecHitSoA.h

```diff
@@ -0,0 +1,15 @@
+#ifndef CUDADATAFORMATS_HGCRECHITSOA_H
+#define CUDADATAFORMATS_HGCRECHITSOA_H 1
```

Comment:
I have changed all guards that refer to the producer to the ```RecoLocalCalo/HGCalRecProducers``` folder. This can be changed later, depending on where the code is put.